tests/Aspire.Hosting.Tests/XunitAttributes.cs

```diff
@@ -0,0 +1,6 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+[assembly: CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
???

Reply:
It means when you run all the tests they don't all try to bind to the same ports :)